src/Silverback.Integration/Messaging/Messages/DefaultMessageHeaders.cs

```diff
@@ -99,5 +99,23 @@ public static class DefaultMessageHeaders
         ///     consumer side to determine the correct key to be used to decrypt the message.
         /// </remarks>
         public const string EncryptionKeyId = "x-encryption-key-id";
+
+        /// <summary>
+        ///     This will be set by the Move error policy and will contain the reason the failed
+        ///     message is being moved.
+        /// </summary>
+        public const string FailureReason = "x-failure-reason";
+
+        /// <summary>
+        ///     This will be set by the Move error policy and will contain the timestamp of the failed
+        ///     message.
+        /// </summary>
+        public const string SourceTimestamp = "x-source-timestamp";
+
+        /// <summary>
+        ///     The header that will be filled with the timestamp of the message consumed from the broker.
+        ///     Depending on the broker this value can be NULL.
+        /// </summary>
+        public const string TimestampKey = "x-message-timestamp";
```

Comment:
Since the mapping is now done per broker (is Kafka specific), this header can be moved back into the Kafka package, right?

Reply:
Yes, right, I moved it back to `KafkaMessageHeaders`.